src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -354,3 +354,133 @@ UicOperatingPeriod/ValidDayBits | | | see (1) below
 This is a string sequence of `0` (inactive days) and `1` (active days).  The
 sequence should contains as many `0/1` as there is days in the period.  The
 first character correspond to the first day of the period.
+
+## ligne_<line_id_md5>.xml
```

Comment:
This is a case (one more?) of trying to make data intended to be read by computer, easy to read by a human at the price of technical complexity... I am not really fond of that.
We could do as you propose with something like
`ligne_<clean_line_id>_<line_id_md5>` with:
- `clean_line_id` being only the alphanumerical characters of the `line_id`
- `line_id_md5` the MD5 checksum of the `line_id` to avoid collisions of the previous rule

Reply:
This file will contains all the routes, trips and stop_times of a particular line. `offre` doesn't really mean anything in the model, does it?

Also, I'm curious about why using `line_code` and not `line_id` since `line_code` is optional?

Reply:
Fixed in 9ecccb0f3807487d2c2fa44b9004fa17769fac96.